hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/utils/TestCompactionUtil.java

```diff
@@ -77,6 +78,11 @@ void beforeEach(Map<String, String> options) throws IOException {
 
     this.table = FlinkTables.createTable(conf);
     this.metaClient = table.getMetaClient();
+    // initialize the metadata table path
+    if (conf.getBoolean(FlinkOptions.METADATA_ENABLED)) {
+      FlinkHoodieBackedTableMetadataWriter.create(table.getHadoopConf(), table.getConfig(),
+          table.getContext(), Option.empty(), Option.empty());
+    }
```

Comment:
This should not be required as the metadata table is automatically created.

Reply:
This is only for the test.